src/sources/journald.rs

```diff
@@ -98,7 +98,7 @@ type Matches = HashMap<String, HashSet<String>>;
 #[derive(Clone, Debug, Default)]
 #[serde(deny_unknown_fields, default)]
 pub struct JournaldConfig {
-    /// Only include entries that appended to the journal after Vector starts reading it.
+    /// Only include entries that appended to the journal after starting.
```

Comment:
"Starting" by itself feels a little too ambiguous because it could imply the "starting" of the operating system itself i.e. "since boot". I'm not really sure how to word this better, though, if we're trying to avoid ambiguous identifiers such as "it."

Reply:
@tobz, how about this?
"Only include entries that appended to the journal after the entries have been read."

Reply:
I updated it to that, let me know if you have any thoughts on it.

Reply:
Works for me.

We might want to wordsmith it later on, but we (the Vector team) probably need a bit more practice of writing comments in this generic way (not saying "Vector") before we can get a grip on how to better express statements like these where Vector/OP are the unspoken referent.